### PR TITLE
Exported test suite so that store-dependent tests can be run from external repositories

### PR DESCRIPTION
**Pull request by thehenrytsai**
Quite heavy-handed on both approach and implementation to get this support going after failing various experimentations....... so open to more elegant alternatives and/or improvements!

### CODE REVIEW

File: tests/handlers/records-write.spec.ts [hunk 1]
```diff
@@ -34,1487 +34,1490 @@ import { RecordsWrite } from '../../src/interfaces/records-write.js';
...
+export function testRecordsWriteHandler(): void {
+  describe('RecordsWriteHandler.handle()', async () => {
+    let didResolver: DidResolver;
+    let messageStore: MessageStore;
+    let dataStore: DataStore;
+    let eventLog: EventLog;
+    let dwn: Dwn;
 
-  describe('functional tests', () => {
+    describe.only('functional tests', () => {
```
Comment (diehuxx) [inline tests/handlers/records-write.spec.ts:52]:
```suggestion
    describe('functional tests', () => {
```

**Review by diehuxx — COMMENTED** [review]
Added one small comment. Here are my broader thoughts.

This is definitely improvement for people developing a custom store, but there's a slight QoL drawback for you and me. We can't do `test.only` or `test.skip`, which will make debugging annoying for us. We can pretty easily workaround by commenting out a few things while we debug. Though I wonder if custom store developers want the ability to run individual tests too. I can't find a graceful way to do that right now, so for the scope of this PR, LGTM.

File: tests/handlers/records-write.spec.ts (hunk 1 above)
Comment (thehenrytsai) [reply tests/handlers/records-write.spec.ts]: Thanks for catching this! Didn't realize `.only` stopped working as I was using it to test my changes quickly. Fixed.

**Review by diehuxx — APPROVED** [review]
niiiiice. lgtm

**Pull request merged**